code/modules/mob/living/carbon/human/life.dm

```diff
@@ -279,22 +279,21 @@
 			ExtinguishMob()
 		for(var/obj/item/clothing/C in get_equipped_items())
 			if(C.wetable)
```

Comment:
I would advise making wetable a function that checks for the wet datum unless it can't be so

Reply:
There needs to exist a var to tell if the cloth can get wet or not, that is what initialize the datum in the clothing or else it will be just a null reference